### PR TITLE
Unity sim

### PR DESCRIPTION
**Pull request by EmilyRobotics**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,10 +0,0 @@
-pip_cache/
```
Comment (porglezomp) [inline .gitignore:1]: Why did you delete the `.gitignore` ??

File: Simulation/robot_sim/Assets/.gitignore [hunk 2]
```diff
@@ -0,0 +1,34 @@
+/[Ll]ibrary/
```
Comment (porglezomp) [inline Simulation/robot_sim/Assets/.gitignore]: This is at too low a level, this has to be in `Simulation/robot_sim/`

**Review by porglezomp — DISMISSED** [review]
The `.gitignore` is in the wrong place, and so you have a lot of garbage files in here that shouldn't be in here.

File: .gitignore (hunk 1 above)
Comment (EmilyRobotics) [reply .gitignore:1]: Fixed!

**Review by EmilyRobotics — COMMENTED** [review]
I fixed it

**Comment by porglezomp** [comment]
You didn't delete all the files that need to be ignored, so they're still here

**Comment by porglezomp** [comment]
We're moving this to https://github.com/FRC1076/Simulation, so this can be closed now.
